datamodel/initdb.d/03_dcsa_im_v3_0.sql

```diff
@@ -634,7 +636,7 @@ CREATE INDEX ON dcsa_im_v3_0.shipment_location (booking_id);
 
 DROP TABLE IF EXISTS dcsa_im_v3_0.port_call_status_code CASCADE;
 CREATE TABLE dcsa_im_v3_0.port_call_status_code (
-    port_call_status_code varchar(4) NOT NULL PRIMARY KEY,
+    port_call_status_type varchar(4) NOT NULL PRIMARY KEY,
```

Comment:
This seems strange to me - I know it is what is mentioned in the Issue - can I just investigate and get back to you...